pkg/model/awsmodel/iam.go

```diff
@@ -21,8 +21,8 @@ import (
 	"sort"
 	"strings"
 
+	awsIam "github.com/aws/aws-sdk-go-v2/service/iam"
```

Comment:
Nit: we should probably switch `awsIam` -> `awsiam` at some stage, it's more go idiomatic I believe